pkg/providers/cloudstack/config/template-md.yaml

```diff
@@ -82,7 +85,7 @@ spec:
               - -E
               - lazy_itable_init=1,lazy_journal_init=1
             filesystem: {{ .cloudstackDiskOfferingFilesystem }}
-            label: data_disk
+            label: {{ .cloudstackDiskOfferingLabel }}
```

Comment:
is this a high priority bugfix for the upcoming release?

Reply:
I don't think so. 
customer can avoid it by setting label as data_disk in yaml.